pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[project]
+name = "awslambdaric"
+version = "2.0.12"
+description = "AWS Lambda Runtime Interface Client for Python"
+authors = [
+    { name = "Amazon Web Services" }
+]
+dependencies = []
+readme = "README.md"
+requires-python = ">= 3.8"
```

Comment:
should we stick with 3.6?
```
python_requires=">=3.6",
```

Reply:
It's technically not supported anymore. I am not sure. We can do either.